README.md

```diff
@@ -81,7 +110,9 @@ body that says "hello".
 
 Your Answer:
 ```js
-// your code here
+$("#greeting").click(function() {
+  $("p").append("hello")
```

Comment:
Yes, this is the correct syntax but I think the question was asking you to append a `<p> tag` to the `body `instead of appending a string to a `<p> tag`. Not a big deal at all though, you understand the concept :)
```

$("#greeting").on("click", function(){
  $("body").append("<p>hello</p>");
});
```